content/sensu-go/5.11/reference/datastore.md

```diff
@@ -79,10 +77,21 @@ With the `PostgresConfig` resource definition saved to a file (for example: post
 sensuctl create --file postgres.yml
 {{< /highlight >}}
 
-At this time, there is no supported method for viewing or deleting a `PostgresConfig` resource.
+At this time, there is no supported method for viewing a `PostgresConfig` resource.
 To update your Sensu PostgreSQL configuration, repeat the `sensuctl create` process shown above.
 You can expect to see PostgreSQL status updates and error messages in the [Sensu backend logs][2] at the `warn` and `error` log levels, respectively.
 
+#### Disabling the PostgreSQL event store
+
+To disable the PostgreSQL event store, use `sensuctl delete` with your `PostgresConfig` resource definition:
+
+{{< highlight shell >}}
+sensuctl delete --file postgres.yml
+{{< /highlight >}}
+
+When disabling the PostgreSQL event store, event data cuts over from PostgreSQL to etcd, resulting in a loss of recent event history.
+No restarts or Sensu backend configuration changes are required to disable the PostgreSQL event store.
+
```

Comment:
Does `sensuctl delete` require 100% match between all resource attributes or just name and namespace?